app/controllers/sessions_controller.rb

```diff
@@ -2,12 +2,13 @@ class SessionsController < ApplicationController
   def new; end
 
   def create
-    user = User.find_by email: params[:session][:email].downcase
-    if user&.authenticate params[:session][:password]
-      log_in user
-      redirect_to user
+    @user = User.find_by email: params[:session][:email].downcase
+    if @user.try(:authenticate, params[:session][:password])
+      log_in @user
+      params[:session][:remember_me] == "1" ? remember(@user) : forget(@user)
```

Comment:
please, move 1 to settings.yml file and call here

Reply:
done